README.md

```diff
@@ -3,27 +3,27 @@ Clair
 
 [![Docker Repository on Quay](https://quay.io/repository/coreos/clair/status "Docker Repository on Quay")](https://quay.io/repository/coreos/clair)
 
-Clair is a container vulnerability analysis service. It provides the list of vulnerabilities that threaten each container and can sends notifications whenever new vulnerabilities that affect existing containers are released.
+Clair is a container vulnerability analysis service. It provides the list of vulnerabilities that threaten each container and can send notifications whenever new vulnerabilities that affect existing containers are released.
 
-We named the project « Clair », which means in French *clear*, *bright*, *transparent* because we believe that it enables users to have a clear insight into the security of their container infrastructure.
+We named the project « Clair », which in French means *clear*, *bright*, *transparent*, because we believe that it enables users to have a clear insight into the security of their container infrastructure.
 
 ## Why should I use Clair?
 
-Clair is a single-binary server that exposes an JSON, HTTP API. It does not require any agent to sit on your containers neither does it need any specific container tweak to be done. It has been designed to perform massive analysis on the [Quay.io Container Registry](https://quay.io).
+Clair is a single-binary server that exposes a JSON HTTP API. It does not require any agent to sit in your containers, nor does it need any specific container tweaks. It has been designed to perform massive analysis on the [Quay.io Container Registry](https://quay.io).
```

Comment:
s/sit/run/
OR
soft suggest:
It does not require any in-container monitoring agent, nor any other container modifications.